src/WebJobs.Script/Binding/NotificationHubBinding.cs

```diff
@@ -21,11 +21,18 @@ internal class NotificationHubBinding : FunctionBinding
             base(config, metadata, access)
         {
             TagExpression = metadata.TagExpression;
+            ConnectionString = metadata.Connection;
```

Comment:
This ConnectionString / Connection interpretation is going to cause confusion for us until we standardize things